modules/nw-ne-ways-to-manage-ingress-traffic.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * ingress/understanding-networking.adoc
+
+[id="nw-ne-ways-to-manage-ingress-traffic_{context}"]
+= Ways to manage ingress controller traffic
+The following table provides an overview of ways that Administrators can manage their Ingress traffic:
+|===
+|Method |Why use?
+
+|Ingress
+|An ingress is a Kubernetes resource that provides some of the same features as routes (which are an OpenShift resource). Ingresses accept external requests and proxy them based on the route. You can only allow certain types of traffic: HTTP, HTTPS and server name identification (SNI), and TLS with SNI. In OpenShift, routes are generated to meet the conditions specified by the ingress object. Ingress managed LB
+Route
+
+|Route
+|Routes provide ingress traffic to services in the cluster. Routes were created before Kubernetes ingress objects and provide more features. Routes provide advanced features that may not be supported by Kubernetes ingress controllers through a standard interface, such as TLS re-encryption, TLS passthrough, and split traffic for blue-green deployments.
```

Comment:
`Routes were created before Kubernetes ingress objects` is misleading statement, indeed the OCP created a concept called Route for the same purpose.